chalk-rust-ir/src/lib.rs

```diff
@@ -221,6 +225,9 @@ pub struct TraitFlags {
     pub non_enumerable: bool,
 
     pub coinductive: bool,
+
+    /// Indicates that a trait is object safe
+    pub object_safe: bool,
```

Comment:
I think we probably want a function `is_object_safe(TraitId)` instead of a flag, since we actually have to do some computation for this and we don't have to have to do that unless we need to.

Reply:
Okay, coming back to this. I realized that you *did* add that. This should just be removed.